### ISSUE TITLE
[BUG] Images in Aspect Box are not sized correctly

### ISSUE THREAD

**Issue by lewisdorigo** [scottish-government-design-system/design-system#113]
## Description of the issue

When an Aspect Box is used within a page layout, images are incorrectly sized.

This is because, in [src/base/objects/layout/_layout.scss](https://github.com/scottish-government-design-system/design-system/blob/7f59084c768264d55461fcc2eb5c790714f9c2b1/src/base/objects/layout/_layout.scss#L12), image height is set to `auto`.

This selector's specificity means it is overriding the height specified in [src/components/aspect-box/_aspect-box.scss](https://github.com/scottish-government-design-system/design-system/blob/7f59084c768264d55461fcc2eb5c790714f9c2b1/src/components/aspect-box/_aspect-box.scss#L38).

## Steps to reproduce the issue
```html
<div class="ds_layout">
    <div class="ds_layout__content">
          <div class="ds_aspect-box  ds_aspect-box--square">
                <img
                     src="//picsum.photos/1280/720/"
                     alt=""
                     class="ds_aspect-box__inner"
                />
          </div>
    </div>
</div>
```
## Expected behaviour
The image shows as a square, cropped to the centre of the image.

## Screenshots
<img width="1392" alt="Screenshot 2024-08-08 at 12 47 24 PM" src="https://github.com/user-attachments/assets/97c67aa6-35f6-472e-b25a-7fb2be46f908">

(Added outlines to aspect box, and to image to show bounding boxes).

**Comment by jsutcliffe** [comment]
Thanks Lewis. A fix for this is currently in testing and will be in the next release.

**Comment by jsutcliffe** [comment]
Fixed in [commit 07e8fb6](https://github.com/scottish-government-design-system/design-system/commit/07e8fb6dfe0efde40ebdf33d9baaf834065574b2), released in [npm package version 2.3.0](https://www.npmjs.com/package/@scottish-government/design-system/v/2.3.0).
